src/sci/sci.cc

```diff
@@ -64,6 +64,9 @@ double SelectedCIMethod::compute_energy() {
         prune_PQ_to_P();
     }
 
+    // Active space PT2 correction
+    full_mrpt2();
```

Comment:
I personally think `full_mrpt2()` should be called in `post_iter_process()`. This may not be common to all `sci` algorithms.

Reply:
I like that change, will modify asap